.github/workflows/ci-staging-deploy.yml

```
@@ -16,119 +16,141 @@ jobs:
   build-earthly:
     permissions: write-all
     uses: ./.github/workflows/build-earthly.yml
-    with:
-      BUILT_EARTHLY_PATH: "./build/linux/amd64/earthly"
-      RUNS_ON: "ubuntu-latest"
-      BINARY: "docker"
-      SUDO: ""
-    secrets: inherit
-
-  tests:
-    name: staging release
+  prepare-release:
+    name: prepare release
     needs: build-earthly
     runs-on: ubuntu-latest
     permissions:
       contents: write
       packages: write
+    outputs:
+      release_tag: ${{ steps.compute-tag.outputs.release_tag }}
+      earthly_next: ${{ steps.read-next.outputs.earthly_next }}
     env:
       FORCE_COLOR: 1
-      EARTHLY_INSTALL_ID: "earthbuild-githubactions"
       GITHUB_USER: "earthbuild"
       EARTHLY_REPO: "earthbuild-staging"
-      BREW_REPO: "homebrew-earthbuild-staging"
       DOCKERHUB_USER: ${{ vars.DOCKERHUB_USERNAME }}
       DOCKERHUB_IMG: "earthbuild-staging"
       DOCKERHUB_BUILDKIT_IMG: "buildkitd-staging"
-      # Used in our github action as the token - TODO: look to change it into an input
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - name: Set up QEMU
-        id: qemu
-        uses: docker/setup-qemu-action@v3
-        with:
-          image: tonistiigi/binfmt:latest
-          platforms: all
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8  #v5.0.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           fetch-depth: 0
+      - name: Retrieve earthbuild from build-earthly job
+        run: |-
```

nit: Unless we really care about that final newline we can just use `|`
```suggestion
        run: |
```